From 7568768b446ce6e1dedf67a61f1d8d2673c54672 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sun, 17 Sep 2017 18:01:14 -0400
Subject: [PATCH 3/5] Add copyright for tests/sha.h tests/sha224-256.c
 tests/sha-private.h

For the discussion with debian-legal, consult the following thread:
https://lists.debian.org/debian-legal/2017/08/msg00003.html
---
 debian/copyright | 40 ++++++++++++++++++++++++++++++++++++++++
 1 file changed, 40 insertions(+)

diff --git a/debian/copyright b/debian/copyright
index a569c389..318e967c 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -19,6 +19,10 @@ Copyright: 2007-2009 Oracle <http://www.oracle.com/>
            2001 Christoph Hellwig
 License: GPL-2
 
+Files: tests/sha.h tests/sha224-256.c tests/sha-private.h
+Copyright: 2011 IETF Trust
+License: TLP-4
+
 Files: debian/*
 Copyright: 2007-2013 Daniel Baumann <daniel.baumann@progress-technologies.net>
            2013 <luk@debian.org>
@@ -57,3 +61,39 @@ License: GPL-2
  .
  The complete text of the GNU General Public License
  can be found in /usr/share/common-licenses/GPL-2 file.
+
+License: TLP-4
+ Copyright (c) 2011 IETF Trust and the persons identified as
+ authors of the code.  All rights reserved.
+ .
+ Redistribution and use in source and binary forms, with or
+ without modification, are permitted provided that the following
+ conditions are met:
+ .
+ - Redistributions of source code must retain the above
+   copyright notice, this list of conditions and
+   the following disclaimer.
+ .
+ - Redistributions in binary form must reproduce the above
+   copyright notice, this list of conditions and the following
+   disclaimer in the documentation and/or other materials provided
+   with the distribution.
+ .
+ - Neither the name of Internet Society, IETF or IETF Trust, nor
+   the names of specific contributors, may be used to endorse or
+   promote products derived from this software without specific
+   prior written permission.
+ .
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+ CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+ NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+ OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+ EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-- 
2.11.0

